.clj-kondo/config.edn

```diff
@@ -0,0 +1,3 @@
+{:linters
+ {:unresolved-var {:level :off} ;; set b/c seems to be triggered by fns exported with tech.datatype.export-symbols/export-symbols fn.
```

Comment:
Wish we didn't have to turn this off, but until we better understand `export-symbols`, and if there are other ways of defining the api, it seems to be caused by that.